### PR TITLE
implement GroupServiceManager

### PR DESCRIPTION
**Pull request by cyjseagull**
1. implement GroupServiceManager

2. add BcosSDK to init the groupManagerService and create client for specified group

### CODE REVIEW

File: src/main/java/org/fisco/bcos/sdk/service/GroupManagerServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+                                        Thread.currentThread().sleep(fetchGroupListIntervalMs);
+                                    } catch (InterruptedException e) {
+                                        logger.warn(
+                                                "GroupManagerSerivce: capature InterruptedException: {}",
+                                                e.getMessage());
+                                        break;
+                                    }
+                                }
+                            }
+                        });
```
Comment (MaggieNgWu) [inline src/main/java/org/fisco/bcos/sdk/service/GroupManagerServiceImpl.java]: Recommend use ScheduledThreadPoolExecutor to manage a schedule task.

File: src/main/java/org/fisco/bcos/sdk/service/GroupManagerServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+        }
+        running.set(false);
+        this.worker.interrupt();
+    }
+
+    /** start the thread to obtain group list information periodically */
+    protected void start() {
+        if (this.worker != null) {
+            logger.warn("GroupManagerService has already been started");
+        }
```
Comment (MaggieNgWu) [inline src/main/java/org/fisco/bcos/sdk/service/GroupManagerServiceImpl.java]: Why not use running to determine whether this service is running?

File: src/main/java/org/fisco/bcos/sdk/service/GroupManagerServiceImpl.java (hunk 1 above)
Comment (cyjseagull) [reply src/main/java/org/fisco/bcos/sdk/service/GroupManagerServiceImpl.java]: done.

File: src/main/java/org/fisco/bcos/sdk/service/GroupManagerServiceImpl.java (hunk 2 above)
Comment (cyjseagull) [reply src/main/java/org/fisco/bcos/sdk/service/GroupManagerServiceImpl.java]: done.

**Pull request merged**